### ISSUE TITLE
'L' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by giumas** [s-weigand/setup-conda#473]
**Describe the bug**
On Windows using:
```
    - name: setup-conda
      uses: s-weigand/setup-conda@v1
      with:
        update-conda: true
        python-version: '3.11'
        conda-channels: 'conda-forge'
```
I get this error:
```
  D:\a\test\test>L "C:\Miniconda\condabin\..\Scripts\conda.exe"   update -y -n base -c defaults conda 
  'L' is not recognized as an internal or external command,
  operable program or batch file.
  Error: The process 'C:\Miniconda\Library\bin\conda.bat' failed with exit code 9009
```

**Expected behavior**
No error

**Desktop:**

- OS: Windows

**Additional context**
A job with the issue: https://github.com/hydroffice/hyo2_abc2/actions/runs/11424333844/job/31784538534
The issue goes away by setting: `update-conda: false` -> https://github.com/hydroffice/hyo2_abc2/actions/runs/11424333842/job/31784538535

**Comment by s-weigand** [comment]
@giumas Thanks for the bug report.

I can reproduce the issue and it seems to caused by `conda.bat` daisy-chaining.
When the [update command](https://github.com/s-weigand/setup-conda/blob/d15429738c460f4ea3d2f609187fd7f177499f58/src/conda_actions.ts#L270-L284) is called it uses `C:\Miniconda\Library\bin\conda.bat` (`C:\Windows\system32\cmd.exe /D /S /C "C:\Miniconda\Library\bin\conda.bat update -y -n base -c defaults conda"`)

```bat
:: Copyright (C) 2012 Anaconda, Inc
:: SPDX-License-Identifier: BSD-3-Clause
@CALL "%~dp0..\..\condabin\conda.bat" %*
```

which in turn calls `C:\Miniconda\condabin\conda.bat`

```bat
:: Copyright (C) 2012 Anaconda, Inc
:: SPDX-License-Identifier: BSD-3-Clause

@IF NOT DEFINED _CE_CONDA @(
  @SET _CE_M=
  @SET "CONDA_EXE=%~dp0..\Scripts\conda.exe"
)
@IF [%1]==[activate]   "%~dp0_conda_activate" %*
@IF [%1]==[deactivate] "%~dp0_conda_activate" %*

@SET CONDA_EXES="%CONDA_EXE%" %_CE_M% %_CE_CONDA%
@CALL %CONDA_EXES% %*

@IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%

@IF [%1]==[install]   "%~dp0_conda_activate" reactivate
@IF [%1]==[update]    "%~dp0_conda_activate" reactivate
@IF [%1]==[upgrade]   "%~dp0_conda_activate" reactivate
@IF [%1]==[remove]    "%~dp0_conda_activate" reactivate
@IF [%1]==[uninstall] "%~dp0_conda_activate" reactivate

@EXIT /B %ERRORLEVEL%
```

Which then ultimately calls `C:\Miniconda\Scripts\conda.exe` with the arguments.

For some reason, it seems that the `@CALL` isn't interpreted correctly and a rogue `L` ends up in the final call (`L "C:\Miniconda\condabin\..\Scripts\conda.exe"   update -y -n base -c defaults conda `), which causes the error.

**Comment by s-weigand** [comment]
@all-contributors please add @giumas for bug

**Comment by s-weigand** [comment]
@giumas I just released a v1.2.3 and it should work again

**Comment by giumas** [comment]
> @giumas I just released a v1.2.3 and it should work again

Yep, I can confirm it! https://github.com/hydroffice/hyo2_abc2/actions/runs/11425933602/job/31788220443
